src/util/adb.js

```diff
@@ -347,14 +347,16 @@ export default class ADBUtils {
   }
 
   async discoverRDPUnixSocket(
-    deviceId: string, apk: string,
+    deviceId: string, apk: string, msg: string,
     {maxDiscoveryTime, retryInterval}: DiscoveryParams = {}
   ): Promise<string> {
     let rdpUnixSockets = [];
 
     const discoveryStartedAt = Date.now();
 
     while (rdpUnixSockets.length === 0) {
+      log.info(`\n\x1b[93m${msg}\x1b[39m\n`);
```

Comment:
Don't bother to make the message yellow for now, I think that it is a good idea, but this should also be checking that the terminal does actually support colors, take care of potential compatibility issues in different operating system terminal colors support etc.

To deal with styling the output we may want to introduce [chalk](https://github.com/chalk/chalk) as a new dependency, but I prefer to deal with it in a separate PR (e.g. because when we introduce a new dependency we have to also update the package-lock.json file, and I prefer to do it separately).

For now let's just call log.info(msg) here.

Reply:
Sure. I will keep the message the normal color.